vehicles-utils/upload-renting-leasings/upload-renting-leasings.py

```
@@ -0,0 +1,453 @@
+        vehicle_fuel_types,
+    ) = get_all_entities()
+
+    for file in os.listdir(PENDING_DIR):
+        if file.endswith(".xlsx"):
+            try:
+                file_path = os.path.join(PENDING_DIR, file)
+                logging.info(f"Procesando archivo: {file}")
+                df = pd.ExcelFile(file_path).parse("RENTINGS")
+                processed_rows, error_rows = [], []
+
+                for index, row in df.iterrows():
```

Ruff (0.8.2)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)